website/docs/getting-started/configure.md

```diff
@@ -4,4 +4,82 @@ sidebar_position: 6
 ---
 # Configuring odo global settings
 
-This doc explains everything in `~/.odo/preference.yaml` file.
\ No newline at end of file
+The global settings for odo can be found in `preference.yaml` file; which is located by default in the `.odo` directory of the user's HOME directory.
+
+Example -
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+defaultValue="Linux"
+values={[
+{label: 'Linux', value: 'linux'},
+{label: 'Windows', value: 'windows'},
+{label: 'Mac', value: 'mac'},
+]}>
+<TabItem value="linux">/home/user/foo/myProject/.odo/preference.yaml</TabItem>
+<TabItem value="windows">C:\\myProject\.odo\preference.yaml</TabItem>
+<TabItem value="mac">/home/user/foo/myProject/.odo/preference.yaml</TabItem>
+</Tabs>
+
+---
+A  different location can be set for the `preference.yaml` by exporting `GLOBALODOCONFIG` in the user environment.
+
+### View the configuration
+To view the current configuration, run `odo preference view`.
+
+```shell
+odo preference view
+```
+Example -
+```shell
+$ odo preference view
+PARAMETER             CURRENT_VALUE
+UpdateNotification
+NamePrefix
+Timeout
+BuildTimeout
+PushTimeout
+Experimental
+Ephemeral
+ConsentTelemetry
```

Comment:
Is the `CURRENT_VALUE` column really empty by default? IMO, there should be default values when user runs that command for the first time.

Reply:
This is something we will need to take up in a different issue. But `CURRENT_VALUE` column can be empty if none of the values are set.